datamodels/2.x/itop-tickets/dictionaries/pt_br.dict.itop-tickets.php

```diff
@@ -239,7 +239,7 @@
 	'Class:ResponseTicketTTR/Interface:iMetricComputer' => 'Time To Resolve~~',
 	'Class:ResponseTicketTTR/Interface:iMetricComputer+' => 'Objetivo baseado em um SLT do tipo TTR',
 
-	'portal:itop-portal' => 'Standard portal~~', // This is the portal name that will be displayed in portal dispatcher (eg. URL in menus)
+	'portal:itop-portal' => 'Portal do usuário~~', // This is the portal name that will be displayed in portal dispatcher (eg. URL in menus)
```

Comment:
When an entry is translated, the "~~" should be removed as they are only here to indicate non translated entries.

```suggestion
	'portal:itop-portal' => 'Portal do usuário', // This is the portal name that will be displayed in portal dispatcher (eg. URL in menus)
```